src/Time/StepChoosers/Increase.hpp

```diff
@@ -0,0 +1,77 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cmath>
+#include <limits>
+#include <pup.h>
+
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Time/StepChoosers/StepChooser.hpp"  // IWYU pragma: keep
+#include "Time/Time.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace Parallel {
+template <typename Metavariables>
+class ConstGlobalCache;
+}  // namespace Parallel
+namespace Tags {
+struct TimeStep;
+}  // namespace Tags
+/// \endcond
+
+namespace StepChoosers {
+
+/// Suggests increasing the step size by a constant ratio.
+template <typename StepChooserRegistrars>
+class Increase : public StepChooser<StepChooserRegistrars> {
+ public:
+  /// \cond
+  Increase() = default;
+  explicit Increase(CkMigrateMessage* /*unused*/) noexcept {}
+  using PUP::able::register_constructor;
+  WRAPPED_PUPable_decl_template(Increase);  // NOLINT
+  /// \endcond
```

Comment:
Maybe it's because it's early, but we do we not need to override the destructor?

Reply:
Why would we?  These classes only contain doubles.

Reply:
Sorry, by `override` I meant do we need to `~Constant() override = default;`?

Reply:
I don't think we need to, no.

Reply:
Also here

Reply:
We've generally been marking all destructors in the inheritance chain as virtual (override), whether we need to or not because the class should always be the most derived. I think especially when serialization is involved this is good practice. Another solution would be to mark the class as `final`. Does that clarify? :)

Reply:
guideline C.128 at https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md suggests to no mark derived destructors with anything...

Reply:
Okay, I now understand why they recommend it. I'm fine with not specifying anything for the destructor. But since these classes should never be a base class let's mark the class 'final'